### PR TITLE
Swap out satori/uuid for google/uuid

### PR DESCRIPTION
**Pull request by davidji99**
Continuation of #182

### CODE REVIEW

**Comment by davidji99** [comment]
@mars Can you review and run all the tests? Thanks!

**Comment by mars** [comment]
Running Hashicorp CI now ⏳and will review too 😄

**Comment by mars** [comment]
@freeformz please checkout this continuation (& hopefully conclusion) of your PR.

File: heroku/validators.go [hunk 1]
```diff
@@ -3,12 +3,17 @@ package heroku
...
-		errors = append(errors, fmt.Errorf("%q is an invalid UUID: %s", k, err))
+// validateUUID matches type terraform.SchemaValidateFunc
+func validateUUID(v interface{}, k string) ([]string, []error) {
+	s, ok := v.(string)
+	if !ok {
+		return nil, []error{fmt.Errorf("%q is an invalid UUID: unable to assert %q to string", k, v)}
 	}
-	return
+	if _, err := uuid.Parse(s); err != nil {
+		return nil, []error{fmt.Errorf("%q is an invalid UUID: %s", k, err)}
```
Comment (freeformz) [inline heroku/validators.go]: Maybe include s in the output as well?

File: heroku/validators.go [hunk 2]
```diff
@@ -3,12 +3,17 @@ package heroku
...
 
-	"github.com/satori/uuid"
+	"github.com/google/uuid"
 )
 
-func validateUUID(v interface{}, k string) (ws []string, errors []error) {
-	if _, err := uuid.FromString(v.(string)); err != nil {
-		errors = append(errors, fmt.Errorf("%q is an invalid UUID: %s", k, err))
+// validateUUID matches type terraform.SchemaValidateFunc
+func validateUUID(v interface{}, k string) ([]string, []error) {
```
Comment (freeformz) [inline heroku/validators.go]: nit:
renaming v => val and k => key would help with understandability here
(I had to look up terraform.SchemaValidateFunc to understand what was going on)

Comment (davidji99) [reply heroku/validators.go]: will do

**Pull request merged**